### ISSUE TITLE
Move all Settings tab at the end

### ISSUE THREAD

**Issue by dannycoulombe** [PolymindApp/client#29]
In Strategy, Component and Dataset the Settings tab is always the first one. It's great for creating new elements but once done, the real value comes from the other tabs (Interface, Source or Data).

We should make those firsts and move Settings at the end. When creating a new project, also open the most important page first (Interface, Source, Data) and if an issue is being raised by the server at saving time, automatically switch to the Settings tab.

![image](https://user-images.githubusercontent.com/10353756/78843168-0f980180-79d0-11ea-8d85-a35b223b94c3.png)

In other words:

1. Make Settings tab last in Strategy, Component and Dataset.
2. Make Settings page open first if user is creating (new), but open new first tab first if item is existing (check App.vue for that).

_* Note: In App.vue, the redirect parameter can also be a function. Look documentation:_
https://router.vuejs.org/guide/essentials/redirect-and-alias.html#redirect

_* Tip: So a good idea could be to detect if the :id parameter is equal to "new" or not and return the proper redirection there._
